### ISSUE TITLE
Ask the user weather to install the aur-package or not when it is already build

### ISSUE THREAD

**Issue by Edenhofer** [rmarquis/pacaur#307]
Thereby one can see how big the final package is. This information can't be derived from the PKGBUILD. It is something the user only knows when the package is alredy assembled.

This can easily be achieved by allowing userinput when invoking "pacman -U /path/to/pkg".

**Comment by rmarquis** [comment]
Pacaur doesn't invoke pacman -U. The whole install process is entirely done through makepkg, unlike other helpers.
Also, doesn't make sense with the "fast workflow" approach.

**Comment by Edenhofer** [comment]
What about an extra option for that?
Since it is quite interesting to know the size of the package prior to installation and one may also change the curse and abord installation.

**Comment by rmarquis** [comment]
Let me rephrase it: implementing such a feature would require extra, unnecessary code which would also be quite error prone, especially when dealing with slit packages. I do agree that having the size of compiled AUR packages would be great, but I'm not ready to deal with all the caveats of such option. And again, doesn't make sense with the fast workflow.

**Comment by Edenhofer** [comment]
I am fine with how it is, it was just an suggestion. I understand your point and must agree that it would add code which is error prone and most of the time useless but nevertheless it would be a quite nice feature.

**Comment by rmarquis** [comment]
To be honest, I feel the only way I can imagine this feature implemented would be by extending the AUR RPC interface (ie, including the size of compiled package in the SRCINFO file... if that make any sense). Anyway, makepkg/pacman do check available space before installing packages, so there is no real technical issue here. A package is too big for the user? Simply uninstall it.
